pyproject.toml

```diff
@@ -0,0 +1,42 @@
+[project]
+name = "epubhv"
+version = "0.3.0"
+description = "make your epub books vertical or horizontal."
+authors = [{ name = "yihong0618", email = "zouzou0208@gmail.com" }]
+
+dependencies = [
+    "beautifulsoup4==4.12.2",
+    "bs4==0.0.1",
```

Comment:
This package is an alias of beautifulsoup and can be removed

Reply:
Thanks! Fixed in the latest commit. Running CI now.